confluence-xml/src/main/java/org/xwiki/contrib/confluence/filter/event/ConfluenceFilteringEvent.java

```diff
@@ -40,6 +42,8 @@ public class ConfluenceFilteringEvent extends AbstractCancelableEvent
 {
     private Collection<Long> disabledSpaces;
 
+    private Map<String, String> renamedSpaces;
```

Comment:
I wouldn't speak in terms of renaming, but in terms of target.

What's more, we could unconditionally save the target for any space, even those we haven't "renamed".
```suggestion
    private Map<String, String> spaceTargets = new HashMap<>();
```